modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1339,10 +1278,10 @@ private static String getPrimitiveType(Schema schema) {
             return "BigDecimal";
         } else if (ModelUtils.isByteArraySchema(schema)) {
             return "ByteArray";
-        } else if (ModelUtils.isFileSchema(schema)) {
```

Comment:
`isFileSchema` method returns true even if schema is binary. 
so in order to detect "binary", place `isBinarySchema` sentence prior to `isFileSchema`.

Reply:
cf. https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java#L461

Reply:
Thanks for the explanation. In OAS3, `file` type (in OAS2) has been deprecated and replaced with `type: string, format: binary` instead.

Reply:
Thanks for reply.
Yes, I known it by https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java#L461 . 

The problem is that return type is "File" from the aspect of performance, for `file` type (in OAS2) and `type: string, format: binary` (in OAS3). cf. https://github.com/OpenAPITools/openapi-generator/issues/994 description.

So I think there're 2 strategies.

1. distinguish BinarySchema and FileSchema
In this case, my patch works.
For backward compatibility, ModelUtils' `isFileSchema` still should detect not only "FileSchema " but also "BinarySchema", as you mention and https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java#L461 says.
Then change type mapping of both `binary` (https://github.com/OpenAPITools/openapi-generator/pull/995/files#diff-f7863d157cb04293ab52ff741f420c0cR146) and `file` (https://github.com/OpenAPITools/openapi-generator/pull/995/files#diff-f7863d157cb04293ab52ff741f420c0cR147) to `ImputStream`

2. handle BinarySchema and FileSchema samely
In this case, we should merge `isFileSchema` method and `isBinarySchema` method into 
`isFileSchemaOrBinarySchema`.

```
else if (ModelUtils.isFileSchemaOrBinarySchema(schema)) {
  return "binary"
}
// delete isFileSchema sentence and isBinarySchema sentence
```

and then keep https://github.com/OpenAPITools/openapi-generator/pull/995/files#diff-f7863d157cb04293ab52ff741f420c0cR146.

I feel like 1 is preferable. 
How do you think?